packages/interface/src/getHoverInformation.js

```diff
@@ -0,0 +1,186 @@
+/**
+ *  Copyright (c) Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the license found in the
+ *  LICENSE file in the root directory of this source tree.
```

Comment:
Ok, so I think I've put the wrong license in here. The original code is under BSD and requires its license to be copied alongside. (I am assuming this port counts as a “redistribution” “with modifications”)

```
Redistribution and use in source and binary forms, with or without modification,
are permitted provided that the following conditions are met:

 * Redistributions of source code must retain the above copyright notice, this
   list of conditions and the following disclaimer.
```